src/ripple/app/tx/impl/Transactor.cpp

```diff
@@ -635,7 +635,7 @@ Transactor::operator()()
 {
     JLOG(j_.trace()) << "apply: " << ctx_.tx.getTransactionID ();
 
-#ifdef BEAST_DEBUG
+#ifdef RIPPLED_DEBUG
```

Comment:
I don't think this macro is actually defined anywhere ? We already have `DEBUG` and `_DEBUG`, but you can add `RIPPLED_DEBUG` here: https://github.com/ripple/rippled/blob/e76f19a6715335bfac2a8f13ba45843cb4869ec2/Builds/CMake/RippledCompiler.cmake#L19

Reply:
I'll use `DEBUG` instead.